assets/css/main.css

```
@@ -170,6 +170,28 @@ body {
   text-decoration: none;
 }
 
+#Explanation{
+  font-size: 12pt; 
+}
+
+#Toggle-Explanation{
+  background: #f1652991;
```

Unfortunately, [RGBA hex is not supported in IE and Edge](https://caniuse.com/#feat=mdn-css_properties_color_alpha_hexadecimal_notation)

Instead, can you use the existing CSS Custom Properties for this, with the Sass variable as the fallback (must come first). This would ensure that this looks normal in dark mode. See below:

```suggestion
  background-color: $theme-primary;
  background-color: var(--theme-primary);
```